src/Containers/FeatureToggle/FeatureToggle.jsx

```diff
@@ -0,0 +1,16 @@
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+const FeatureToggle = ({ isEnabled, children }) => (isEnabled ? children : null);
+
+FeatureToggle.propTypes = {
+  isEnabled: PropTypes.bool,
+};
+
+function isFeatureEnabled({ features }, featureName) {
```

Comment:
What if set a default parameter of `{ features = [] }`? Otherwise tests fail if this container is actually implemented in an un-connected test.

```suggestion
function isFeatureEnabled({ features = [] }, featureName) {
```